.gitignore

```diff
@@ -40,7 +53,8 @@ src/icons/skin/build-icons/node_modules/
 src/icons/skin/build-icons/icons/*.png
 
 # CMake build directory
-build/
+build*
+cmake-build*
```

Comment:
>Considering we are using Cmake as build system, it's OK with Cmake build directory?

It is not clear for me about the reasons behind this change. Could you elaborate?

Reply:
>There is no file build that should not be ignored by VCS, so I removed the slash for a wider match. Also add cmake-build for a regular output directory.

That line (`build/`) was insisted (very strongly) by someone that claims to be the de facto cmake build directory... however that claim is disputable. To avoid wasting time with the complaints we just allowed it and IMO we shouldn't bother it anymore.